docs/app/documentation/component-docs/time/time-docs.component.html

```diff
@@ -50,8 +50,12 @@ <h2>Disabled State</h2>
 
 <h2>Internationalization</h2>
 <description>
-  The time component uses a custom service called TimeI18nLabels to give aria-labels to the buttons and inputs.
-  Inspect the HTML elements to see the Spanish aria-labels.
+    The time component uses a custom service called TimeI18nLabels to give aria-labels to the buttons and inputs.
+    Inspect the HTML elements to see the Spanish aria-labels.
+    The time component uses also a custom service called TimeI18n to give possibility, to change naming of meridian
```

Comment:
The time component also uses a custom service called TimeI18n to allow the possibility of changing the name of the meridian modifier. In the example below, the german vorm. and nach. are shown.